diff -Nru a/qtbase/mkspecs/linux-iMX28/qmake.conf b/qtbase/mkspecs/linux-iMX28/qmake.conf
--- a/qtbase/mkspecs/linux-iMX28/qmake.conf	1970-01-01 01:00:00.000000000 +0100
+++ b/qtbase/mkspecs/linux-iMX28/qmake.conf	2019-02-15 17:59:33.878682949 +0100
@@ -0,0 +1,18 @@
+#
+# qmake configuration for building with arm-linux-gnueabi-g++
+#
+
+MAKEFILE_GENERATOR      = UNIX
+CONFIG                 += incremental
+QMAKE_INCREMENTAL_STYLE = sublib
+
+include(../common/linux.conf)
+include(../common/gcc-base-unix.conf)
+include(../common/g++-unix.conf)
+
+QMAKE_CFLAGS     += -mcpu=arm926ej-s -march=armv5te -marm -msoft-float -funroll-loops
+QMAKE_CXXFLAGS   += -mcpu=arm926ej-s -march=armv5te -marm -msoft-float -funroll-loops
+QMAKE_LFLAGS     += -mcpu=arm926ej-s -march=armv5te -marm -msoft-float -funroll-loops
+
+#Helps the board find where the libraries are.
+load(qt_config)
diff -Nru a/qtbase/mkspecs/linux-iMX28/qplatformdefs.h b/qtbase/mkspecs/linux-iMX28/qplatformdefs.h
--- a/qtbase/mkspecs/linux-iMX28/qplatformdefs.h	1970-01-01 01:00:00.000000000 +0100
+++ b/qtbase/mkspecs/linux-iMX28/qplatformdefs.h	2019-02-15 17:59:33.886682949 +0100
@@ -0,0 +1,40 @@
+/****************************************************************************
+**
+** Copyright (C) 2016 The Qt Company Ltd.
+** Contact: https://www.qt.io/licensing/
+**
+** This file is part of the qmake spec of the Qt Toolkit.
+**
+** $QT_BEGIN_LICENSE:LGPL$
+** Commercial License Usage
+** Licensees holding valid commercial Qt licenses may use this file in
+** accordance with the commercial license agreement provided with the
+** Software or, alternatively, in accordance with the terms contained in
+** a written agreement between you and The Qt Company. For licensing terms
+** and conditions see https://www.qt.io/terms-conditions. For further
+** information use the contact form at https://www.qt.io/contact-us.
+**
+** GNU Lesser General Public License Usage
+** Alternatively, this file may be used under the terms of the GNU Lesser
+** General Public License version 3 as published by the Free Software
+** Foundation and appearing in the file LICENSE.LGPL3 included in the
+** packaging of this file. Please review the following information to
+** ensure the GNU Lesser General Public License version 3 requirements
+** will be met: https://www.gnu.org/licenses/lgpl-3.0.html.
+**
+** GNU General Public License Usage
+** Alternatively, this file may be used under the terms of the GNU
+** General Public License version 2.0 or (at your option) the GNU General
+** Public license version 3 or any later version approved by the KDE Free
+** Qt Foundation. The licenses are as published by the Free Software
+** Foundation and appearing in the file LICENSE.GPL2 and LICENSE.GPL3
+** included in the packaging of this file. Please review the following
+** information to ensure the GNU General Public License requirements will
+** be met: https://www.gnu.org/licenses/gpl-2.0.html and
+** https://www.gnu.org/licenses/gpl-3.0.html.
+**
+** $QT_END_LICENSE$
+**
+****************************************************************************/
+
+#include "../linux-g++/qplatformdefs.h"
